.ci_support/linux_64_MSVC_HEADERS_VERSION14.29.30133TOOLCHAIN_COMBINED14.29.16.11clang_version15.0.7.yaml

```diff
@@ -1,7 +1,5 @@
 CHOST_BASE:
 - x86_64-pc-windows-msvc
-CL_VERSION:
-- 19.29.30139
```

Comment:
It seems that https://github.com/conda-forge/clang-win-activation-feedstock/pull/18/commits/45b2f376dbc9495d832195af8bc4d74e772b253f has affected whether `CL_VERSION` gets picked up for linux/osx. This looks like it should still be there IMO, as its used in `install-pkg.sh`.

Reply:
Fixed in https://github.com/conda-forge/clang-win-activation-feedstock/pull/18/commits/1673641387e1f24add184ee37d770ba8a34c1fdd